### PR TITLE
Saves queries by day

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/5050

### CODE REVIEW

File: queries.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (jgsqware) [inline queries.json]: Is that file for testing purpose? If yes would it be better to put it in a tests folder?

**Review by jgsqware — COMMENTED** [review]
Only comments

File: queries.json (hunk 1 above)
Comment (pipo02mix) [reply queries.json]: Added by mistake. Rmeoved

**Review by jgsqware — APPROVED** [review]
LGTM

**Pull request merged**